packages/kusama/src/__snapshots__/peopleKusama.e2e.test.ts.snap

```diff
@@ -2,6 +2,34 @@
 
 exports[`Kusama People > adding a registrar as root from the relay chain works > call add registrar with wrong origin 1`] = `
 [
+  {
+    "data": {
+      "dispatchInfo": {
+        "class": "Mandatory",
+        "paysFee": "Yes",
+        "weight": {
+          "proofSize": "(rounded 1500)",
+          "refTime": "(rounded 260000000)",
```

Comment:
Locally, this is the `refTime` the test's execution returns (`260000000`); but in CI this fails as it expects `270000000`.

@xlc have you seen this happen before? I might just redact this one line.